node/src/accountant/payment_adjuster/adjustment_runners.rs

```diff
@@ -0,0 +1,254 @@
+            initialize_payment_adjuster(now, cw_service_fee_balance_minor, 12345678);
+
+        let result = subject.adjust_accounts(
+            &mut payment_adjuster,
+            vec![payable_1.clone(), payable_2.clone()],
+        );
+
+        assert_eq!(
+            result,
+            vec![
+                AdjustedAccountBeforeFinalization::new(
+                    payable_1.analyzed_account.qualified_as.bare_account,
+                    payable_1.weight,
+                    expected_proposed_balance_1
+                ),
+                AdjustedAccountBeforeFinalization::new(
+                    payable_2.analyzed_account.qualified_as.bare_account,
+                    payable_2.weight,
+                    expected_proposed_balance_2
+                )
+            ]
+        )
+    }
+
+    fn weighted_payable_setup_for_surplus_test(
+        n: u64,
+        initial_balance_minor: u128,
+    ) -> WeightedPayable {
+        let mut account = make_weighed_payable(n, initial_balance_minor);
+        account
+            .analyzed_account
+            .qualified_as
+            .payment_threshold_intercept_minor = 3_000_000_000;
+        account.analyzed_account.qualified_as.creditor_thresholds =
+            CreditorThresholds::new(1_000_000_000);
+        account
+    }
+
+    #[test]
+    fn means_equal_requested_money_after_dsq_in_previous_iteration_to_return_capped_accounts() {
+        let subject = ServiceFeeOnlyAdjustmentRunner {};
+        let cw_service_fee_balance_minor = 10_000_000_000;
+        let mut payable_1 = weighted_payable_setup_for_surplus_test(111, 5_000_000_000);
+        payable_1.analyzed_account.disqualification_limit_minor = 3_444_333_444;
+        let mut payable_2 = weighted_payable_setup_for_surplus_test(222, 5_000_000_000);
+        payable_2.analyzed_account.disqualification_limit_minor = 3_555_333_555;
+        let expected_proposed_balance_1 = 3_444_333_444;
+        let expected_proposed_balance_2 = 3_555_333_555;
+
+        test_surplus_incurred_after_disqualification_in_previous_iteration(
+            subject,
+            payable_1,
+            payable_2,
+            cw_service_fee_balance_minor,
+            expected_proposed_balance_1,
+            expected_proposed_balance_2,
+        )
+    }
+
+    #[test]
+    fn means_become_bigger_than_requested_after_dsq_in_previous_iteration_to_return_capped_accounts(
+    ) {
+        let subject = ServiceFeeOnlyAdjustmentRunner {};
+        let cw_service_fee_balance_minor = 10_000_000_000;
+        let mut payable_1 = weighted_payable_setup_for_surplus_test(111, 5_000_000_000);
+        payable_1.analyzed_account.disqualification_limit_minor = 3_444_333_444;
+        let mut payable_2 = weighted_payable_setup_for_surplus_test(222, 4_999_999_999);
+        payable_2.analyzed_account.disqualification_limit_minor = 3_555_333_555;
+        let expected_proposed_balance_1 = 3_444_333_444;
+        let expected_proposed_balance_2 = 3_555_333_555;
```

Comment:
If we create variables for payable balances then:

```
let untaken_cw_balance = payable_1_balance + payable_2_balance + 1;
```

You may prefer to assign the expected balances to the disqualification limits.